### DISCUSSION TITLE
Error message when data is being fetched from my connector

### DISCUSSION THREAD

**Discussion by HendrikHahn1996** [eclipse-tractusx/tractusx-edc#1651] [Q&A]
Hello community,

I am currently running into error and I don't know what to do.

So, if someone tries to request the catalog in order to fetch data from my connector it returns the following error message: 

`[
  {
    "message": "org.eclipse.edc.http.spi.EdcHttpClientException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target",
    "type": "BadGateway",
    "path": null,
    "invalidValue": null
  }
]`

I am able to fetch data from other connectors and if i call the '**api/v1/dsp**' route with the URL of my connector everythings working fine. 
Also the connection is secured using the SSL certificate. 

Does anyone have an idea? 

Thanks in advance!

**Comment by rafaelmag110** [comment]
Hi!
Is the certificate you are using with your TLS a self-signed one? If that is the case I would suggest to get a certificate that is issued by a widely known CA.

**Reply by HendrikHahn1996** [reply]
Hey,

the certificate is not a self-signed one. Its an official and valid certificate.

**Reply by rafaelmag110** [reply]
Understood. Then i'd suggest you to try and find which specific request from your or the other participant EDC is failing to validate the certificate. This error is not specific to the tractusx-edc, but how the host is setup. There are plenty of resources online that can help you find and fix this issue.

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
